src/app/components/home/home-top-banner.component.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+
+.home-header-container {
+  background: linear-gradient($color-cerise-600, $color-primary-cerise);
+  color: $white;
+  display: flex;
+  flex-flow: row-reverse;
+
+  @media screen and (max-width: $md-device-width) {
+    flex-flow: column;
+  }
+
+  .home-header-map-image {
+    padding-right: 0;
+    width: 426px;
+
+    @media screen and (max-width: $md-device-width) {
+      margin: 0 auto;
+    }
+
+    @media screen and (max-width: 426px) {
+      width: 100%;
+    }
+  }
+
+  .home-header-info-container {
+    padding: 40px;
+    text-align: left;
+    width: calc(100% - 426px);
+
+    @media screen and (max-width: $md-device-width) {
+      margin: 0 auto;
+      max-width: 800px;
+      padding-left: 10px;
+      padding-right: 10px;
+      text-align: center;
+      width: auto;
+    }
+
+    .home-header-info-title {
+      @include wrap-long-words;
+      color: $white;
+      text-shadow: 0 0 3px rgba(0, 0, 0, 0.75);
+    }
+
+    .home-header-info-subtitle {
+      padding-bottom: 0.2em;
```

Comment:
Properties should be ordered padding-top, padding-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)